content-src/lib/utils.js

```diff
@@ -79,5 +79,18 @@ module.exports = {
   getRandomFromTimestamp(percent, site) {
     const n = (site.lastVisitDate || site.bookmarkDateCreated || Math.round(Math.random() * 100)) % 100;
     return n <= percent * 100;
+  },
+
+  // before: data-l10n-id={this.props['data-l10n-id']}
+  // after:  {...l10n(this.props)}
+  l10n(props) {
```

Comment:
I wrote this helper because I found myself typing `data-l10n-id={this.props['data-l10n-id']}` a lot.  It works well because components in Activity Streams are well designed and generally one component only needs a single `data-l10n-id` to be passed.